### DISCUSSION TITLE
Are there any plans to support other architectures of linux

### DISCUSSION THREAD

**Discussion by LeLeF** [mandiant/capa#781] [Q&A]
I noticed that so far the elf module only supports x86 architecture. Are there any plans to support other architectures of linux? Such as ARM/SPARC/PowerPC/MIPS and more. Thank you for your answer.

**Comment by mr-tz** [comment]
While support for other architectures is possible, it's not a priority for us at the moment.
With our current file support we cover the majority of files we (and hopefully most analysts) encounter. I think we should focus on making the tool better for those. Of course, if that changes, we'll be happy to adapt.
